Nova-LMS-Project/src/com/lms/dao/BookDao.java

```diff
@@ -0,0 +1,21 @@
+package com.lms.dao;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+import com.lms.model.Book;
+
+public interface BookDao {
```

Comment:
Most of my comments for `AuthorDao` and `AuthorDaoImpl` also apply to `BookDao` and `BookDaoImpl`.